### ISSUE TITLE
Error when using play button from Ubuntu sound indicator

### ISSUE THREAD

**Issue by skyghis** [tiliado/nuvola-app-plex#4]
````
Error during call of Nuvola.actions.emit: Function 'Nuvola.actions.emit' failed. file:///usr/share/nuvolaplayer3/js/main.js:154: TypeError: null is not an object (evaluating 'elm.dispatchEvent')

The application might not function properly.
````
Bug steps : 
1. Start Nuvolaplayer Plex
2. Navigate to a page where music is not available (e.g. home)
3. Click the play button from Ubuntu sound indicator

This bug is caused by the play event triggered even if the `player.setCanPlay` is set to `false`.

**Comment by fenryxo** [comment]
Hello. Thanks for your report. It seems to be a generic MPRIS issue as it occurs also with GNOME Shell Media Player exension and with other services. I've created a ticket tiliado/nuvolaplayer#136 for it and added it at the end of the August 2015 queue tiliado/nuvolaplayer.project#5.

**Comment by skyghis** [comment]
Should I add defensive code to prevent this issue from `nuvola-app-plex` side ?

**Comment by fenryxo** [comment]
No, it's a responsibility of Nuvola Player core to ensure disabled actions cannot be activated. I thought this check is already done by GLib's Action class, but this assumption is obviously wrong and I have to fix my code then ;-)

**Comment by skyghis** [comment]
I confirm the fix on v3.0.0+201511042023.413b716
